src/modules/generateRandomNumber.js

```diff
@@ -7,7 +7,13 @@
  * @return {number} A random 4-digit number
  */
 function generateRandomNumber() {
-  /* Write your code here */
+  const randomNumber = Math.floor(Math.random() * (10000 - 1) + 1);
+
+  if (new Set(randomNumber.toString()).size !== 4) {
```

Comment:
The check for duplicate digits is correct, but ensure that the generated number is always 4 digits long and does not start with 0.